openfga/v1/openfga_service.proto

```diff
@@ -672,34 +677,40 @@ service OpenFGAService {
     };
   }
 
-  // StreamedListObjects is a streaming variation of the ListObjects API (see ListObjects for more info).
   rpc StreamedListObjects(StreamedListObjectsRequest) returns (stream StreamedListObjectsResponse) {
     option (google.api.http) = {
       post: "/stores/{store_id}/streamed-list-objects",
       body: "*"
     };
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "[EXPERIMENTAL] Streaming variation of the ListObjects API (see ListObjects for more info)"
       tags: [
         "Relationship Queries"
       ]
       operation_id: "StreamedListObjects"
     };
   }
 
-  // ListObjects lists all of the object ids for objects of the provided type that the given user
-  // has a specific relation with.
   rpc ListObjects(ListObjectsRequest) returns (ListObjectsResponse) {
     option (google.api.http) = {
       post: "/stores/{store_id}/list-objects",
       body: "*"
     };
 
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
+      summary: "[EXPERIMENTAL] Get all object ids of the given type that the given user has a given relation with"
       tags: [
         "Relationship Queries"
       ]
       operation_id: "ListObjects"
+      description:
+        "The ListObjects API returns a list of all the object ids of the given type that the given user "
+        "has a given relation with. To achieve this, both the store tuples and the authorization model are used.\n"
+        "Path parameter `store_id` as well as body parameters `type`, `relation` and `user` are all required.\n"
+        "An `authorization_model_id` may be specified in the body. If it is, it will be used to decide the underlying implementation used. "
+        "If it is not specified, the latest authorization model ID will be used.\n"
+        "You may also specify `contextual_tuples` that will be treated as regular tuples.\n"
```

Comment:
Actually I am a bit confused with the contextual tuples here myself - when contextual tuples are specified, does it mean that it will be listed (i.e., returned as there is a tuple)? If so, why would someone use this?

Reply:
It behaves the same way as Check. It doesn't necessarily mean that they will be returned directly.

As to the use case, we don't know yet, we'll find out if it's useful or not 🤷‍♀️

Reply:
Ahh...I see.